app/admin/category/new/template.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+  <form class="add__new-form">
```

Comment:
Make this form into a `simple-form` with an `onsubmit`action handler. You can add `class` attributes to ember components just like HTML elements!